packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/parentPropertyGridComponent.tsx

```diff
@@ -57,7 +65,7 @@ export class ParentPropertyGridComponent extends React.Component<IParentProperty
                     noDirectUpdate={true}
                     onSelect={(value) => {
                         const nodeAsTransform = node as TransformNode;
-                        if (value < 0) {
+                        if (typeof value === "number" && value < 0) {

```

Comment:
Don't you want to check `typeof value !== "number" || value < 0` instead? Else, for a non number value (only other possible value is "string" I guess?), you will take the `else` branch and do `sortedNodes[value as number]`, which won't work if value is not a number.

Reply:
Good catch, thanks!